Cargo.lock

```diff
@@ -6794,9 +6794,9 @@ dependencies = [
 
 [[package]]
 name = "openssl"
-version = "0.10.60"
+version = "0.10.64"
```

Comment:
Please bump dependencies in separate PR, so that we can easily revert any bad changes.

Reply:
My fault, seems this is not a dep version bump